data/repositories.toml

```diff
@@ -667,4 +667,5 @@ repositories = [
   'github.com/dbidwell94/vanilact',
   'github.com/boa-dev/boa',
   'github.com/globaleaks/GlobaLeaks'
+  'github.com/maddypie/PyCalc'
```

Comment:
Hey @maddypie, thanks for your contribution. Can you please add a `,` on the previous line to make this a valid TOML?